soccer/gameplay/skills/dribble.py

```diff
@@ -88,13 +103,27 @@ def aimed(self):
         angle = self.robot.angle - (self.pos - self.robot.pos).angle()
         return angle < .1 and angle > -.1
 
+    def on_enter_setup(self):
+        self.move_vector = main.ball().pos - self._pos
+        self.move_point = self._pos + self.move_vector + (
+            self.move_vector.normalized() * .2)
```

Comment:
I might be wrong, but I don't think that's the same thing. The `.2` is the flat distance past the ball, not an additionally 20% of the move vector. For both small and large vectors, it will make a significant difference.